native/packages/react-native-bpk-component-button-link/src/layout/BpkButtonLinkWrapper.ios.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
```

Comment:
Is it really worth splitting these two up instead of just using a single file with one `Platform.select`? Seems like we are ending up with a lot of duplication here

Reply:
the two wrappers are different for instance the andorid one requires a static `TouchableNativeFeedback.SelectableBackgroundBorderless()` and I like the separated strucutre a lot, so at least for now, I would keep it like this

Reply:
I don't agree, but let's agree to disagree and leave it :)